root/etc/e-smith/templates.metadata/etc/httpd/conf.d/nextcloud.conf

```diff
@@ -0,0 +1 @@
+DELETE=1
```

Comment:
If removed here the nextcloud.con file can be resurrected by an RPM update. I'd rather expand it to a noop config